models/datasets/generic/ds__diagnoses.sql

```diff
@@ -0,0 +1,35 @@
+select
+    e.id as encounter_id,
+    p.id as patient_id,
+    diagnosis.id as diagnosis_id,
+    diagnosis.name as diagnosis,
+    ed.datetime as diagnosis_datetime,
+    p.first_name,
+    p.last_name,
+    p.display_id,
+    date_part('year', age(ed.datetime, p.date_of_birth)) as age,
+    initcap(p.sex::text) as sex,
```

Comment:
Should already be formatted.
```suggestion
    sex,
```